.github/ISSUE_TEMPLATE/bug.yml

```
@@ -0,0 +1,123 @@
+body:
+  # Greeter message
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting a bug! Please first check the boxes below to ensure that your bug report meets our requirements.
+  # Requirements checks
+  - type: checkboxes
+    attributes:
+      options:
+        - label: I'm using the latest version of Natron (not required but recommended)
+        - label: I've restarted Natron and the issue persists
+          required: true
+        - label: I've run Natron via the [command line](https://natron.readthedocs.io/en/rb-2.4/devel/natronexecution.html) and the issue persists
+          required: true
+        - label: I've followed the [contributing guidelines](https://github.com/NatronGitHub/Natron/blob/RB-2.4/CONTRIBUTING.md) to the best of my understanding
+          required: true
+        - label: My issue is not on the [issue tracker](https://github.com/NatronGitHub/Natron/issues?q=is%3Aissue+type%3Abug) and is not a duplicate of a forum thread
+          required: true
+        - label: This bug is reproducible
+          required: true
+        - label: This issue is not a feature request or a pull request
+          required: true
+  # Natron & OS versions
+  - type: input
+    id: natron-version
+    attributes:
+      label: Natron version
+      description: Natron version/commit (they can be retrieved from the about window or with `natron --about`)
+      placeholder: "e.g. Natron 2.4.0"
+    validations:
+      required: true
+  - type: input
+    id: os-version
+    attributes:
+      label: Operating system
+      description: The name and version of your operating system
+      placeholder: "e.g. macOS 10.15 Catalina"
+    validations:
+      required: true
+  # System (hardware) info
+  - type: textarea
+    attributes:
+      label: System specs
+      render: markdown
+      placeholder: |
+        RAM: 15.6 GiB
+        CPU: 8 x Intel Core i7-7700HQ CPU @ 2.80GHz
+        GPU: NVIDIA GeForce GTX 1070/PCle/SSE2
+      description: |
+        Please input the following (if you know them):
+          - Your system's RAM
+          - The number, model, and type of your CPU(s)
+          - The model and type of your GPU
+    validations:
+      required: false
+  # Did the user install via the official installer?
+  - type: checkboxes
+    attributes:
+      label: Did you install Natron using the official installer?
+      description: If you installed Natron via another method, please add your installation method in the \"additional details\" section at the bottom of this form
+      options:
+        - label: Yes, I used the official installer
+        - label: No, I installed from a binary archive
+        - label: No, I compiled Natron from sources
+        - label: No, I installed Natron via another method
+    validations:
+      required: true
+  # At which location did the user install Natron?
+  - type: textarea
+    attributes:
+      label: At which location did you install Natron?
```

Most people will install Natron to the default location.  I don't think this should be required and should be reworded accordingly
```suggestion
      label: If you installed to a custom (non-default) location please include it here:
```